app/components/post_list/post_list.js

```diff
@@ -392,11 +393,13 @@ export default class PostList extends PureComponent {
     };
 
     flatListScrollToIndex = (index) => {
-        this.flatListRef.current.scrollToIndex({
-            animated: false,
-            index,
-            viewOffset: 0,
-            viewPosition: 1, // 0 is at bottom
+        this.animationFrameInitialIndex = requestAnimationFrame(() => {
+            this.flatListRef.current.scrollToIndex({
+                animated: false,
+                index,
+                viewOffset: 0,
+                viewPosition: 1, // 0 is at bottom
+            });
```

Comment:
I wonder if this could magically fix [MM-21276](https://mattermost.atlassian.net/browse/MM-21276) 🤞

Reply:
this is just to give it time